code/game/area/areas/ruins/lavaland.dm

```diff
@@ -14,34 +15,39 @@
 	icon_state = "dk_yellow"
 
 /area/ruin/powered/snow_biodome
+	name = "\improper Snow Biodome"
 	icon_state = "dk_yellow"
 
 /area/ruin/powered/gluttony
+	name = "\improper Gluttony Ruin"
```

Comment:
putting "ruin" in the names makes them feel weird when its deadly sins

Reply:
That's fair, I'll be a bit more creative.